docs/apis/metrics.md

```diff
@@ -157,6 +157,55 @@ public class MyMapper extends RichMapFunction<Long, Integer> {
 }
 {% endhighlight %}
 
+#### Meter
+
+A `Meter` measures an average throughput. An occurrence of an event can be registered with the `markEvent()` method. Occurrence of multiple events at the same time can be registered with `markEvent(long n)` method.
```

Comment:
Is `markEvent` a bit too long ? Maybe we can use `mark` instead. I find Counter and Histogram follow dropwizard's function name.